engine/src/main/battlecode/instrumenter/bytecode/InstrumentingMethodVisitor.java

```diff
@@ -153,6 +152,9 @@ public void visitMaxs(int maxStack, int maxLocals) {
 
         boolean anyTryCatch = tryCatchBlocks.size() > 0;
 
+        // must be called before addDebugHandler() so debug methods are profiled properly
+        addEnterMethodHandler();
```

Comment:
so i guess even if instrumenting is disabled we inject the instrumenter code?

(again this is only my commments so i can figure out what happens here)

Reply:
Not anymore